Jenkinsfile

```diff
@@ -87,7 +87,7 @@ builds.ath = {
             checkout scm
             withMavenEnv(["JAVA_OPTS=-Xmx1536m -Xms512m",
```

Comment:
Again potentially misleading.

Reply:
That is now called withJdkEnv and only configures the JDK for non-linux environments.